fdbbackup/backup.actor.cpp

```diff
@@ -2400,6 +2460,74 @@ ACTOR Future<Void> describeBackup(const char *name, std::string destinationConta
 	return Void();
 }
 
+// If restoreVersion is invalidVersion or latestVersion, use the maximum or minimum restorable version respectively for
+// selected key ranges. If restoreTimestamp is specified, any specified restoreVersion will be overriden to the version
+// resolved to that timestamp.
```

Comment:
"If restoreTimestamp is specified, any specified restoreVersion will be overriden to the version
 // resolved to that timestamp."

I think we should reject the request because it means user didn't know what version they want to restore to.

Reply:
What I meant was:
1) If user specifies restoreTimestamp OR restoreVersion, we accept it;
2) If user specifies both restoreTimestamp AND restoreVersion, we should reject it and remind user that only one option should be provided. If we really want to accept both options at the same time, we will have to validate restoreTimestamp is converted to the same restoreVersion user provided. Silently dropping or overriding an option is a bad idea, which invite operation mistakes.